python/import_util.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime, date, time, timedelta
+import json as js
+from mage.export_import_util.parameters import Parameter
+import mgp
+
+
+def convert_from_isoformat(property):
+    if not isinstance(property, str):
+        return property
+
+    if str.startswith(property, Parameter.DURATION.value):
+        duration_iso = property.split("(")[-1].split(")")[0]
+        parsed_time = datetime.strptime(duration_iso, "%H:%M:%S.%f")
+        return timedelta(
+            hours=parsed_time.hour,
+            minutes=parsed_time.minute,
+            seconds=parsed_time.second,
+            microseconds=parsed_time.microsecond,
+        )
+    elif str.startswith(property, Parameter.LOCALTIME.value):
+        local_time_iso = property.split("(")[-1].split(")")[0]
+        return time.fromisoformat(local_time_iso)
+    elif str.startswith(property, Parameter.LOCALDATETIME.value):
+        local_datetime_iso = property.split("(")[-1].split(")")[0]
+        return datetime.fromisoformat(local_datetime_iso)
+    elif str.startswith(property, Parameter.DATE.value):
+        date_iso = property.split("(")[-1].split(")")[0]
+        return date.fromisoformat(date_iso)
+    else:
+        return property
```

Comment:
Is this leftover? I mean if it is nothing of `Parameter.*.value`, do you need to raise Exception here or return property?